lib/std/target.zig

```diff
@@ -77,6 +77,8 @@ pub const Target = struct {
             }
         };
 
+        // TODO do we really need these constants? Versions 19h1 and 19h2 share
+        // the same NTDDI version, we cannot encode them both in this format
```

Comment:
Yes, they are used when specifying the windows version in the `-target` option, like this: `-target x86_64-windows.win7..win10_19h1`

Reply:
I mean it could be a simple enum, these encoded values do not appear to be used anywhere

Reply:
https://github.com/ziglang/zig/pull/6157/files#diff-d6c6ee4e32d6d1b30e8e7e467e9125c7R264-R265

Reply:
Yes version is encoded here, but why have u32 when all tags do fit in u8? I don't really care since it works, i only wanted to point out that it is overcomplicated

Reply:
Idk, one might say it is overcomplicated to transform those values into a u8 rather than use the 32-bit standard values that microsoft defines. I don't think it's a big deal either way. Feel free to make an official proposal to change it